dotnetUno5/.vscode/launch.json

```diff
@@ -0,0 +1,62 @@
+{
+  // Use IntelliSense to find out which attributes exist for C# debugging
+  // Use hover for the description of the existing attributes
+  // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
+  "version": "0.2.0",
+  "configurations": [
+    {
+      // Use IntelliSense to find out which attributes exist for C# debugging
+      // Use hover for the description of the existing attributes
+      // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
+      "name": "Debug Local",
+      "type": "coreclr",
+      "request": "launch",
+      "preLaunchTask": "build-skia-gtk",
+      // If you have changed target frameworks, make sure to update the program path.
+      "program": "${workspaceFolder}/__change__.Skia.Gtk/bin/Debug/net8.0/__change__.Skia.Gtk.dll",
+      "args": [],
+      "env": {
+        "DOTNET_MODIFIABLE_ASSEMBLIES": "debug"
+      },
+      "cwd": "${workspaceFolder}/__change__.Skia.Gtk",
+      // For more information about the 'console' field, see https://aka.ms/VSCode-CS-LaunchJson-Console
+      "console": "internalConsole",
+      "stopAtEntry": false
+    },
+    {
+      "name": "Torizon ARMv8",
+      "type": "coreclr",
+      "request": "launch",
+      "program": "~/app/__change__.Skia.Gtk",
+      "cwd": "~/app/",
+      "stopAtEntry": false,
+      "console": "internalConsole",
+      "env": {
+        "DOTNET_MODIFIABLE_ASSEMBLIES": "debug",
+        //"GDK_BACKEND": "x11",
+        //"GDK_GL": "gles",
+        "UNO_FULLSCREEN": "true"
+      },
+      "args": [],
+      "pipeTransport": {
+          "pipeCwd": "${workspaceFolder}",
+          "pipeProgram": "ssh",
+          "pipeArgs": [
+              "-T",
+              "-q",
+              "-p",
+              "${config:torizon_debug_ssh_port}",
+              "-i",
+              "${workspaceFolder}/.conf/id_rsa", // ssh key path
+              "-o",
+              "StrictHostKeyChecking=no",
+              "-o",
+              "UserKnownHostsFile /dev/null",
+              "${config:torizon_run_as}@${config:torizon_ip}" // user@device
+          ],
+          "debuggerPath": "/vsdbg/vsdbg"
+      },
+      "preLaunchTask": "deploy-torizon-arm64"
+    }
+  ]

```

Comment:
I think Torizon ARMv7 is missing

Reply:
yeah, this is on purpose. From the telemetry data we have, the use of armv7 is too small to justify new work on it. I'm not saying we're going to abandon it or deprecate it, but we're going to add support now on demand, I think it's fair.

Reply:
I think is ok for now. Probably in the future we will also move it to "Toradex supported", and so we can add this armv7 one in the future

Reply:
The armv7 tasks that come from `assets/common.json`  will not be automatically added to the template right?

Reply:
the common tasks will be added, but I believe that they all hide, so I do not see a problem